ui/app/styles/components/selectable-card.scss

```diff
@@ -5,6 +5,31 @@
   grid-gap: 2rem;
 }
 
+.selectable-card-container-graph {
```

Comment:
is this class applied to the `SelectableCard` component or the `SelectableCardContainer` component? If it's applied to the container, it should probably live in a separate scss file called `selectable-card-container.hbs` to keep our styles organized.

Reply:
also, what do you think about renaming this class `.selectable-card-container.has-grid` or something similar? i noticed you used that pattern with `.change-metric-icon.is-decrease` below which was awesome, because it clearly separates the component from a separate state!

Reply:
Yep, good call.  I'll make another .scss file
And, yes, I like that idea!  I changed it to `.selectable-card-container.has-grid`.